react/graphql/products.graphql

```diff
@@ -0,0 +1,79 @@
+query Products(
+  $category: String
+  $collection: String
+  $specificationFilters: [String]
+  $orderBy: String
+  $from: Int
+  $to: Int
+  $hideUnavailableItems: Boolean = false
+) {
+  products(
+    category: $category
+    collection: $collection
+    specificationFilters: $specificationFilters
+    orderBy: $orderBy
+    from: $from
+    to: $to
+    hideUnavailableItems: $hideUnavailableItems
+  ) @context(provider: "vtex.search-graphql") {
+    cacheId
+    productId
+    productName
+    productReference
+    description
+    link
+    linkText
+    brand
+    brandId
+    specificationGroups {
+      name
+      specifications {
+        name
+        values
+      }
+    }
+    items {
```

Comment:
You're missing `variations` from `items`. I don't if that's the case, but shouldn't this be inside [store-resource](https://github.com/vtex-apps/store-resources)?

Reply:
I think we should use the one already in store-resource

Reply:
Awesome! Did not think about that hahaha makes perfect sense

Reply:
There are some arguments that I need to use in order to perform this query using the one from `store-resources`, I'll add them there

Reply:
Solved here: https://github.com/vtex-apps/store-resources/pull/68